server/views/referrals/new/additionalInformation/show.njk

```diff
@@ -64,14 +64,25 @@
           value: formValues.formattedAdditionalInformation or referral.additionalInformation,
           errorMessage: errors.messages.additionalInformation
         }) }}
-
-        {{ govukButton({
+        <div class="govuk-button-group">
+          {{ govukButton({
           attributes: {
             'data-testid': 'additional-information-submit-button'
           },
           text: "Continue"
         }) }}
-      </form>
+          {{ govukButton({
+          attributes: {
+            'data-testid': 'additional-information-skip-button'
+          },
+          classes: "govuk-button--secondary",
+          text: "Skip this section",
+          name: "skip",
+          value: "true"
+        }) }}
+        </form>
+      </div>
```

Comment:
These 2 need to be swapped and probably worth neatening up the indentation of the `govukButton` properties.